src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Mashine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator почав працювати.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator закінчив працювати.");

```

Comment:
The messages in the `doWork` and `stopWork` methods should be in English. Consider changing them to something like "Excavator started working." and "Excavator stopped working." to meet the checklist requirement of using English and making messages informative.